CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 1.14.0-beta2 - 2022-09-05
+### Fixes
```

Comment:
```suggestion
### Fixed
```

Reply:
we use Fixes in other releases, and yes i can shoot a pr with one changelong for 1.14.0, later this week

Reply:
https://github.com/nextcloud/mail/blame/main/CHANGELOG.md once.

https://keepachangelog.com/en/1.0.0/ for the specs.